docs/resources/Auto_Moderation.md

```diff
@@ -0,0 +1,221 @@
+# Auto Moderation
+
+Auto Moderation is a per [guild](#DOCS_RESOURCES_GUILD/) feature which allows the configuration of a set of rules
+to trigger based on some criteria, e.g. a message contains a specific keyword. Certain actions, such as blocking a message,
+can be setup to automatically execute whenever a rule is triggered.
```

Comment:
why are there line breaks here?

Reply:
fixed.

Reply:
To make it easier to work with in source, it will render without line breaks on the site.

Reply:
I guess we don't do that on other pages in the doc 🤔 I feel like we should though to make it easier to work with, I was thinking same reason you'd format your code to not have long lines. But i'll remove it to keep it consistent with rest of docs